### ISSUE TITLE
NoClassDefFoundError when building a new Ceylon project

### ISSUE THREAD

**Issue by azolotko** [ceylon/ceylon-ide-intellij#130]
I've created a new Ceylon project and added a new module to it. When I choose Build/Make Project the Messages Make tab contains the following:
```
Information:4/11/16, 8:04 PM - Compilation completed with 1 error and 0 warnings in 372ms
Error:ceylon: java.lang.NoClassDefFoundError: com/redhat/ceylon/cmr/impl/InvalidArchiveException
```

```
IntelliJ IDEA 2016.1.1
Build #IC-145.597, built on March 29, 2016
JRE: 1.8.0_77-b03 x86_64
JVM: Java HotSpot(TM) 64-Bit Server VM by Oracle Corporation
```

ceylon-ide-intellij @ df30dab21a7e78df0e395213ca3d813d182be7d1

Not quite sure, maybe something's wrong with my setup. I also get lots of 
`WARNING: Cannot add resolver com.redhat.ceylon.cmr.impl.JSUtils - java.lang.NoClassDefFoundError: com/redhat/ceylon/cmr/resolver/javascript/JavaScriptResolver` when starting a new instance of IntelliJ via Debug 'CeylonIDEA' configuration.

**Comment by bjansen** [comment]
What version of the plugin are you using (you can see it in Settings > Plugins), and which OS please?

**Comment by bjansen** [comment]
Oh I see, you are building it from master, sorry :)

**Comment by bjansen** [comment]
I think this is the same problem I fixed in 6d33e39c605a64f93e56f3be50de648a0201d85e, could you please test again on the current master?

**Comment by azolotko** [comment]
I've just tested at 9c7103dcd96, it works perfectly. Thanks!
